packages/api-catalog-spa/src/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.scss';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import '@patternfly/react-core/dist/styles/base.css';
+import 'swagger-ui-react/swagger-ui.css';
+import 'graphql-voyager/dist/voyager.css';
+
+if ( !(window as any).OpAuthHelper ) {
```

Comment:
Can you please add the ts for the window function in global declaration.
Ref: https://github.com/1-Platform/idea-hub/blob/main/src/types.d.ts

Reply:
Updated